python/cudf/cudf/core/column/timedelta.py

```diff
@@ -306,6 +306,39 @@ def as_timedelta_column(self, dtype: Dtype) -> TimeDeltaColumn:
             return self
         return libcudf.unary.cast(self, dtype=dtype)
 
+    def find_and_replace(
+        self,
+        to_replace: ColumnBase,
+        replacement: ColumnBase,
+        all_nan: bool = False,
+    ) -> TimeDeltaColumn:
+        if not isinstance(to_replace, TimeDeltaColumn):
```

Comment:
It would be nice if we could share this function implementation with DatetimeColumn since they're effectively identical aside from the column type.

Reply:
Do you want me to pack this into a method in our internal utilies and call that utility function from these two column methods?

Reply:
That would be ideal, yes.

Reply:
Done 👍